### PR TITLE
Bump pylint to 3.0.0a7, update changelog

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description

Blocked by #8929

### CODE REVIEW

File: script/.contributors_aliases.json [hunk 1]
```diff
@@ -189,6 +189,10 @@
     "mails": ["anjsimmo@gmail.com", "a.simmons@deakin.edu.au"],
     "name": "Andrew J. Simmons"
   },
+  "antonioglez-23@hotmail.com": {
+    "mails": ["antonioglez-23@hotmail.com"],
+    "name": "Antonio Glez 23"
```
Comment (Pierre-Sassoulas) [inline script/.contributors_aliases.json:194]: Not sure about the name here but there's two "Antonio", we need to distinguish them

Comment (jacobtylerwalls) [reply script/.contributors_aliases.json:194]: Who is the other one? (It's not [this](https://github.com/pylint-dev/astroid/commit/0361eb460fc92da6ac3c69637cc6a8698b509d88) one?)

Comment (Pierre-Sassoulas) [reply script/.contributors_aliases.json:194]: I need to look in detail the link between github account and email to be sure (not doable with only the gît log)

Comment (Pierre-Sassoulas) [reply script/.contributors_aliases.json:194]: Commit come from #8721 and #8728, I'm going to fix this accordingly

**Review by jacobtylerwalls — APPROVED** [review]

**Comment by jacobtylerwalls** [comment]
Thank you 🙏!

**Pull request merged**